### PR TITLE
add database week1 homework

### PR DESCRIPTION
**Pull request by mehmetekinci**

### CODE REVIEW

File: Week1/index.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+      gnp FLOAT,
+      gnp_old FLOAT,
+      local_name VARCHAR(254),
+      government_form VARCHAR(254),
+      head_of_state VARCHAR(254),
+      capital INT,
+      PRIMARY KEY (id)
+      );`;
+
+  connection.connect();
```
Comment (roxana-lafuente) [inline Week1/index.js]: It's always a good idea to add validation on everything that could go on. And connecting to the DB is one of those places where tons of things can go wrong: user does not exist, wrong password, database does not exist, etc

File: Week1/index.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+6          shows answer of "What are the names of all the cities in the
+                             Netherlands?"
+7          shows answer of "What's the population of Rotterdam?"
+8          shows answer of "What's the top 10 countries based on surface area?"
+9          shows answer of "What's the top 10 cities with the highest population?"
+10         shows answer of "What's the population of the world ?"
+  `);
+};
+
+const queries = [
```
Comment (roxana-lafuente) [inline Week1/index.js]: All queries look good, great job! :)

File: Week1/index.js (hunk 1 above)
Comment (mehmetekinci) [reply Week1/index.js]: hey @roxana-lafuente . Thank you for your helpful review. you're right, I also have to do that but I tried to upload homework as soon as possible. I will work on it.

File: Week1/index.js [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+  {
+    query: `select name,population from countries where population > 8000000;`,
+    question: 1,
+  },
+  {
+    query: `select name from countries where name like '%land%';`,
+    question: 2,
+  },
+  {
+    query: `select name,population from cities where population >500000 and population < 1000000;`,
```
[the remark this answers is not in the document]
Comment (mehmetekinci) [reply Week1/index.js]: I am interested  'between' keyword. thank you for advise and helpful feedback.

**Pull request closed without merging**